components/Article/DataTag/index.module.scss

```diff
@@ -0,0 +1,26 @@
+.data-tag {
```

Comment:
Can the class names be `camelCse`?

---

cc @manishprivet could you search for StyleLint configs that ensure camelcase? (Also, could you check if Stylelint is working et all? Somehow linting passed on this https://github.com/nodejs/nodejs.org/pull/5236#discussion_r1167789534